### ISSUE TITLE
Cannot run unit tests on 752

### ISSUE THREAD

**Issue by pokrakam** [SAP/code-pal-for-abap#461]
> 💡 Code pal for ABAP follows the [Clean ABAP](https://github.com/SAP/styleguides/blob/main/clean-abap/CleanABAP.md). If the issue relates to coding style, please submit it [here](https://github.com/SAP/styleguides/issues). 

**Actual Behavior**
Trying to unit test Code Pal, every unit tests dumps on my 752 NPL Developer system 
<img width="705" alt="image" src="https://user-images.githubusercontent.com/25539927/133643303-0318a9dd-486e-4312-a574-f97849f9273a.png">
Not sure if this is specific to my system or 752, I cannot investigate this as it really screws up my system such that I cannot open CL_CI_SCAN after it happened and cannot log on until the system is restarted. Several "Server resources exhausted" messages in the system log. Perhaps someone can test this on another 752 system to see whether it's NPL or 752 in general?

**Expected Behavior**
Unit tests run.

**Comment by lucasborin** [comment]
Hm, that is strange. Can you please get the values of these two tables?
* levels
* tokens

**Comment by pokrakam** [comment]
Here they are:
```
Row  DEPTH  LEVEL  STMNT  FROM  TO  NAME                         TYPE
=====================================================================
1    1      0      0      1     6   CODE_PAL_FOR_ABAP_UNIT_TEST  P


Row  LEVEL  STRUC  FROM  TO  NUMBER  COLONROW  TROW  COLONCOL  TCOL  PREFIXLEN  TYPE  TERMINATOR  ENHMT
=======================================================================================================
1    1      1      1     2   1       0         1     0         16    0          K     .           0
2    1      2      3     3   2       0         2     0         19    0          K     .           0
3    1      3      4     6   3       0         3     0         18    0          C     .           0
4    1      4      7     14  4       0         5     0         17    0          K     .           0
5    1      5      15    17  5       0         6     0         14    0          C     .           0
6    1      4      18    18  6       0         7     0         8     0          K     .           0
```
I think I may have found a clue in the debugger, this is where it all goes off the rails:
<img width="757" alt="image" src="https://user-images.githubusercontent.com/25539927/133671593-988ebd73-a5c5-41cf-b364-7a1c7664dde9.png">

<img width="483" alt="image" src="https://user-images.githubusercontent.com/25539927/133671722-5537363b-5882-4a00-8855-02452250d3a9.png">

Since the ABAP Unit runner keeps going it crashes here for each test and just kills the whole system.

**Comment by lucasborin** [comment]
Can you please set this parameter to `abap_false`?
In parallel, I am looking for a 7.52 here. 

![image](https://user-images.githubusercontent.com/5233413/133675365-403bcc54-184a-4241-962d-05eccc847cac.png)

**Comment by pokrakam** [comment]
No joy, P_NO_CLASSIFICATION doesn't exist on 752. I note this is handled specifically straight after the piece of code you gave:
```ABAP
    CATCH SYSTEM-EXCEPTIONS dyn_call_meth_param_not_found = 1.
      CREATE OBJECT result TYPE (class_type) PARAMETER-TABLE parameters.
    ENDCATCH.

    IF sy-subrc = 1.
      DELETE parameters WHERE name = 'P_NO_CLASSIFICATION'.
      CREATE OBJECT result TYPE (class_type) PARAMETER-TABLE parameters.
    ENDIF.
  ENDMETHOD.
```

It's low priority, mostly curious whether it was my system or 752 in general. I can always use my A4H if need be.

**Comment by lucasborin** [comment]
I found a 7.52 here, and yeah, the bug is there. Let me debug it. :)

**Comment by lucasborin** [comment]
Ps: 7.57 and 7.40 are both working fine.

**Comment by lucasborin** [comment]
@pokrakam: In the `Y_REF_SCAN_MANAGER_DOUBLE`, please change the `CREATE_TRDIR` to:

```abap
  METHOD create_trdir.
    SELECT SINGLE * FROM trdir INTO @result WHERE name = 'Y_REF_SCAN_MANAGER_DOUBLE=====CP'.
  ENDMETHOD.
```
It will solve the issue.

**Comment by pokrakam** [comment]
Yep, works, thanks!

**Comment by lucasborin** [comment]
Actually, it is optional in the `cl_ci_source_include=>feed( )` and the uts works without it. I will open a PR removing it.

**Comment by lucasborin** [comment]
Fix available under branch `1.16.0`.
